.storybook/config.js

```diff
@@ -0,0 +1,8 @@
+import { configure } from '@storybook/react';
+
+function loadStories() {
+    const context = require.context('../src', true, /__stories__\/.+\.js$/);
+    context.keys().map(context);
```

Comment:
I was confused by this for a bit.

1. The result of `context.keys().map()` is unused, so I guess this is performing a side effect per iteration. If possible, consider using `forEach` instead of `map` so that's clearer to the reader?
2. Since `context` is a function, could we name it something with a verb that explains what it does?

Reply:
Ah, good catch - this was a straight copypasta from [the webpack wiki](https://github.com/webpack/docs/wiki/context#context-module-api)

Reply:
re: naming, I think there is value in using the name that is used within all webpack examples (`context`).